ha/const.py

```diff
@@ -132,6 +134,14 @@
 BASE_WAIT_TIME = 5
 NODE_STOP_TIMEOUT = 300 # 300 sec to stop single node
 
+IEC_MAPPING_DICT = {
```

Comment:
Please refer to R1 implementation where we have config file (iem_ha.json) for this purpose. I think we can implement similar in R2.

Reply:
Ok. Will check.